.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/devcontainers/go:1-1.21-bullseye
+FROM mcr.microsoft.com/devcontainers/go:1-1.22-bullseye
```

Comment:
why are we using a Microsoft golang container?

Reply:
I'd guess simply because that's what's suggested in the devcontainer documentation/examples.     My understanding is there's nothing really special about the devcontainer image(s) so we could replace that with anything.